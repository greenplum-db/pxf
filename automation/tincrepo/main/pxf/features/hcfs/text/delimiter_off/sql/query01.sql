-- start_ignore
-- end_ignore
-- @description query01 tests that delimiters are disabled when table creation has delimiter 'off'
--

select * from hcfs_text_delimiter_off;
