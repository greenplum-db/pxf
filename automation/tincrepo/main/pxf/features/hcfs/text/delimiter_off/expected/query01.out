-- start_ignore
-- end_ignore
-- @description query01 tests that delimiters are disabled when table creation has delimiter 'off'
--
select * from hcfs_text_delimiter_off;
                 text_blob                 
-------------------------------------------
 first line
 second line
 third line
 no delimiter, in this file
 commas (,) should be ignored
 a total of 6 lines should be reported ,,,
(6 rows)

